docs/src/manual.md

```diff
@@ -309,3 +308,202 @@ The resulting dual model is
     && con & \in FakeDualCone(3)\\
 \end{align}
 ```
+
+## Advanced
+
+### KKT Conditions
+
+The KKT conditions are a set of inequalities for which the feasible soltion is equivalent to the optimal solution of an optimization problem, as long as strong duality holds and constraint qualification rules such as Slater's are valid. The KKT is used in many branches of optimization and it might be interesting the write them programatically.
```

Comment:
"soltion" missing u

Reply:
and `the write them` --> `to write them`?